jena-arq/src/main/java/org/apache/jena/riot/web/HttpMethod.java

```diff
@@ -16,30 +16,36 @@
  * limitations under the License.
  */
 
-package org.apache.jena.sparql.core;
+package org.apache.jena.riot.web;
 
-import org.apache.jena.graph.Graph ;
-import org.apache.jena.graph.Node ;
+import java.util.Locale;
 
-/** Test using a view-generating dataset */  
-public class TestGraphOverDatasetMem extends AbstractTestGraphOverDatasetGraph
-{
-    @Override
-    protected DatasetGraph createBaseDSG()
-    { 
-        DatasetGraph dsg = DatasetGraphFactory.create() ;
-        return new DatasetGraphViewGraphs(dsg) ;
-    }
-    
-    @Override
-    protected Graph makeDefaultGraph(DatasetGraph dsg)
-    {
-        return dsg.getDefaultGraph() ;
+public enum HttpMethod {
```

Comment:
Some git artifact that is think deleting TestGraphOverDatasetMem is related by a move/rename to adding HttpMethod.